src/FormatNumber.elm

```diff
@@ -1,4 +1,7 @@
-module FormatNumber exposing (format)
+module FormatNumber exposing
+    ( format
+    , parse
+    )
```

Comment:
Does Elm Format do this? I thought it would be better like:

```suggestion
module FormatNumber exposing (format, parse)
```

Reply:
Yes it is from elm format. I am new to Elm so, just used it to format before commit.

Reply:
Excellent then : )